src/forEachChild.ts

```diff
@@ -345,18 +345,18 @@ export function forEachChild(node: any, visitor: (node: SyntaxKind) => SyntaxKin
 }
 
 function visitNode(
-  node: SyntaxKind,
+  node: any,
   parentNode: any,
-  visitor: (node: SyntaxKind, parentNode: SyntaxKind) => SyntaxKind
+  visitor: (node: any, parentNode: any) => SyntaxKind
```

Comment:
What about 'any'?

Reply:
the typings is incorrect, but I don't have to fix it atm.╮(￣▽￣)╭